app/views/site/_study_settings_form.html.erb

```diff
@@ -19,7 +19,7 @@
   </div>
   <div class="form-group row">
     <div class="col-md-4">
-      <%= f.label :embargo, 'No Data Download Until?' %>&nbsp;<span class='fas fa-question-circle' data-toggle='tooltip' title='Prevent data file downloads until date specified below (leave blank to allow downloads)' data-placement='right'></span><br />
+      <%= f.label :embargo, 'Data Embargo Date' %>&nbsp;<span class='fas fa-question-circle' data-toggle='tooltip' title='Downloads will be prevented for everyone except study owners & shared users.  The embargo will expire on the date specified (leave blank to allow downloads)' data-placement='right'></span><br />
```

Comment:
Embargo is the act of banning (downloads in this case) so I believe we're asking for a "Data Embargo End Date" or "Data Release Date"?

Reply:
I like `Data Release Date` - that seems much more explicit.